docs/specifications/tests/README.md

```diff
@@ -33,7 +33,7 @@ project. All the details are in the [Master Test Plan](MasterTestPlan.md).
 ## Document Hierarchy
 
 1. Each Test Level described in [Master Test Plan](MasterTestPlan.md)
-   should be linked directly to the correct level.md document.
+   should be linked directly to the correct level document (README.md).
```

Comment:
I don't think it makes sense to point out the filename without the directory here either. Maybe the definition of the term could be highlighted or something?

Reply:
I'll try, but please suggest better wording if mine is not good enough. Please note that this PR is **ONLY** about changing the file names. Other updates better fit in a new PR.